sci-geosciences/gpsbabel/gpsbabel-1.6.0.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop qmake-utils
+
+MY_PV="${PV//./_}"
+MY_P="${PN}_${MY_PV}"
+
+DESCRIPTION="GPS waypoints, tracks and routes converter"
+HOMEPAGE="https://www.gpsbabel.org/ https://github.com/gpsbabel/gpsbabel"
+SRC_URI="
+	https://github.com/${PN}/${PN}/archive/${MY_P}.tar.gz
+	doc? ( https://www.gpsbabel.org/style3.css -> gpsbabel.org-style3.css )"
+LICENSE="GPL-2"
```

Comment:
per skel.ebuild LICENSE should be one line below, in the same block as `SLOT`

Reply:
Done in the new commit, and moved RESTRICT to the end of the same block.